src/parameters.toml

```diff
@@ -1,2826 +1,2877 @@
 
 [SB2006_raindrops_terminal_velocity_coeff_aR]
 value = 9.65
 type = "float"
-description = "Parameter of the raindrops terminal velocity equation [m/s]. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $a_R$ in the raindrops terminal velocity equation (m s⁻¹). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
 [SB2006_raindrops_terminal_velocity_coeff_bR]
 value = 10.3
 type = "float"
-description = "Parameter of the raindrops terminal velocity equation [m/s]. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $b_R$ in the raindrops terminal velocity equation (m s⁻¹). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
 [SB2006_raindrops_terminal_velocity_coeff_cR]
 value = 600
 type = "float"
-description = "Parameter of the raindrops terminal velocity equation [1/m]. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $c_R$ in the raindrops terminal velocity equation (m⁻¹). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
 [SB2006_ventilation_factor_coeff_av]
 value = 0.78
 type = "float"
-description = "Constant in ventilation factor equation for computing the evaporation rate of rain. Unitless. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $a_v$ in the ventilation factor equation for rain evaporation (unitless). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
 [SB2006_ventilation_factor_coeff_bv]
 value = 0.308
 type = "float"
-description = "Constant in ventilation factor equation for computing the evaporation rate of rain. Unitless. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $b_v$ in the ventilation factor equation for rain evaporation (unitless). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
-[SB2006_rain_evaportation_coeff_alpha]
+[SB2006_rain_evaporation_coeff_alpha]
 value = 159
 type = "float"
-description = "Constant in fallspeed relation for computing the evaporation rate of rain. Unitless. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $\\alpha$ in the fallspeed relation for rain evaporation (unitless). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
-[SB2006_rain_evaportation_coeff_beta]
+[SB2006_rain_evaporation_coeff_beta]
 value = 0.266
 type = "float"
-description = "Constant in fallspeed relation for computing the evaporation rate of rain. Unitless. From Seifert and Beheng, 2006. DOI: 10.1007/s00703-005-0112-4."
+description = "Coefficient $\\beta$ in the fallspeed relation for rain evaporation (unitless). Source: Seifert and Beheng (2006), DOI: 10.1007/s00703-005-0112-4."
 
 [SB2006_raindrops_size_distribution_coeff_N0_min]
-value = 2.5e5
+value = 250000.0
```

Comment:
I would prefer to keep `2.5e5` rather than `250000.0`. And below, `2e7` rather than `20000000.0`. Unless this is some convention we are trying to follow